testdriver/testplan.py

```diff
@@ -50,6 +53,10 @@ def __init__(self, exec_data, test_type, args=None):
 
   def setOptions(self, options):
     self.options = options
+    try:
+      self.icu_version = options.icu_version
+    except:
+      logging.warn('NO ICU VERSION SET')
```

Comment:
praise: the incremental improvements along the way are nice

Reply:
Ok